### ISSUE TITLE
Build for Ubuntu 22.04

### ISSUE THREAD

**Issue by tomhughes** [zerebubuth/openstreetmap-cgimap#254]
Can we get a launchpad build for jammy aka 22.04 please?

**Comment by mmd-osm** [comment]
22.04 comes with a bunch of new library and compiler versions. One change in boost is currently triggering an assertion failure in one of the test cases. This used to work in 20.04 with the following fix https://github.com/zerebubuth/openstreetmap-cgimap/commit/44300e6d54e31491e443b313da2afe7411e4cc34 , but is now failing again. That's quite a PITA. I think  will need to add some manual filtering to only let digits, comma and spaces pass.

**Comment by mmd-osm** [comment]
@tomhughes : a quick question on „errol“, our dev server. I replaced all of boost::format by fmt, which is a transition step to using std::format in c++20.

Now, to avoid everything falling apart, the libfmt-dev package would need to be deployed first. As I don’t have an account on Errol, I can’t check if that’s already the case.

**Comment by tomhughes** [comment]
Don't know if it was already installed but I've pushed https://github.com/openstreetmap/chef/commit/683b98ff859ad7c9db40647ca417890129e2c7ce to make sure it is.

**Comment by mmd-osm** [comment]
Thanks! I think this all worked fine, errol now reports "CGImap 0.8.7-pre" as current version.

**Comment by mmd-osm** [comment]
In the last week, I removed some more old boost code, updated autotools makros, added some missing test cases, removed the old Vagrant-based setup, fixed an issue identified by Coverity, and set FMT_HEADER_ONLY as default to avoid strange linker errors in some environments and updated the README file.

I've uploaded another *test* PPA here: https://launchpad.net/~mmd-osm/+archive/ubuntu/cc5edd52-5b26-4240-a5d8-0e495b680bb7
Based on https://github.com/zerebubuth/openstreetmap-cgimap/tree/ubuntu/jammy

As I don't have much left on my todo list, I hope we can release 0.8.7 fairly soon.

**Comment by mmd-osm** [comment]
Release 0.8.7 is out, PPA is available here: https://launchpad.net/~mmd-osm/+archive/ubuntu/cgimap-807-jammy 

I'm closing here...

**Comment by mmd-osm** [comment]
I've noticed a small glitch in the current build, I forgot to merge the version update from 0.8.7-pre to 0.8.7 (commit: https://github.com/zerebubuth/openstreetmap-cgimap/commit/8fdbf50b8f84cba864dbca2914c577247fe1bb25). Now the "generator" string contains the extra "-pre" in there: "CGImap 0.8.7-pre (886 spike-08.openstreetmap.org)". 

As  this is somewhat embarrassing, I've triggered a new build now: https://launchpad.net/~mmd-osm/+archive/ubuntu/cgimap-807-jammy-v2 

PPAs should be available shortly once they have been published. The updated PPA version number "0.8.7-**2**~jammy1" hopefully supersedes the existing PPA in the osmadmins repo when copying over the new PPA .
